### ISSUE TITLE
Annotating the spec requirements for testing

### ISSUE THREAD

**Issue by edwardsph** [solid/webid-profile#41]
Looking at https://github.com/solid/webid-profile/issues/40, I see loads of clearly articulated requirements coming into the spec which is great for testing. Can I make a request that, perhaps soon after those changes are incorporated, the spec gets annotated with the relevant terms from https://github.com/solid/vocab/blob/specification-terms/spec.ttl and described here: https://github.com/solid-contrib/specification-tests#annotations. This will enable us to create a test suite for this spec which directly links test results to the relevant specification requirements.

**Comment by VirginiaBalseiro** [comment]
Yes, we intend to describe the requirements and others concepts in RDFa once the spec is a bit more stable.

**Comment by jeff-zucker** [comment]
Great idea @edwardsph.  You might want to take a look at https://github.com/jeff-zucker/solid-load-profile which needs a bit of updating but should implement all of the conformance rules.

**Comment by jeff-zucker** [comment]
I have been speaking with @melvincarvalho about the advantages of using JSON-LD data islands instead of RDFa.  This approach puts all the RDF in one place rather than intermixing it with HTML.  A `GET Accept text/turtle` will produce the same results as it would on RDFa but it is possible to patch and much cleaner to read and write.  Basically humans read the HTML and machines read the JSON-LD and there is no intermixing of the two.  Sarven, sorry if I'm thinking of abandoning the RDFa camp.  Is there any interest in following this approach in our spec?  @edwardsph could this work as well for you in the test-suite?

**Comment by edwardsph** [comment]
Thanks, the test harness can load raw RDF or HTML+RDFa but not data islands so I'm pleased with the outcome

**Comment by VirginiaBalseiro** [comment]
Closed by #98
